solution/backend/regulations/tests/test_site_configuration.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import pytest
+from regulations.models import SiteConfiguration
+
+
+@pytest.mark.django_db
+class SiteConfigurationTest(unittest.TestCase):
+    def test_site_configuration_str(self):
+        SiteConfiguration.objects.all().delete()
```

Comment:
is this something that needs to be done before each test or what you are doing for just this test?  Use set up and tear down methods to ensure the clean database or a populated one
https://docs.pytest.org/en/7.1.x/how-to/xunit_setup.html